modules/op-configuring-pull-request-capabilities-in-GitHub-interceptor.adoc

```diff
@@ -0,0 +1,13 @@
+// Ths module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: CONCEPT
+[id="op-configuring-pull-request-capabilities-in-GitHub-interceptor_{context}"]
+= Configuring pull request capabilities in GitHub Interceptor
+
+With GitHub Interceptor, you can create logic that validates and filters GitHub webhooks. For example, you can validate the webhook’s origin and filter incoming events based on specified criteria. When you use GitHub Interceptor to filter event data, you can specify the event types that Interceptor has to accept in a field. 
```

Comment:
```suggestion
With GitHub Interceptor, you can create logic that validates and filters GitHub webhooks. For example, you can validate the webhook’s origin and filter incoming events based on specified criteria. When you use GitHub Interceptor to filter event data, you can specify the event types that Interceptor can accept in a field. 
```
- Not sure about this, but should this sentence express requirement or capability. If the defined event types are required after being defined, then use `must` rather than `has to`; if the defined types are optional and not required, then use `can`.